### ISSUE TITLE
load few reults!!!

### ISSUE THREAD

**Issue by kenpokas** [zachomedia/apple-music-webplayer#103]
my friend please can you fix this unbelievable web load more than 5 reults?more less 20-50 or maybe more...it would be greattttttttttttttttt dude!!!Merry Christmas my friend!!!

**Comment by zachomedia** [comment]
@kenpokas Thank you for your suggestion, can you point me to which part of the app is currently loading 5 results at a time? I can tweak it assuming the Apple Music APIs support a larger load per request.

**Comment by kenpokas** [comment]
woowwwwwwww....thanks so much where answer me!!!!!!!and thanks so muchhhhhhhhhhhhhhh where make this unbelievable web increase unlimited albums and songs!!!!my friend the only where miss for your amazingggggggggggggggggg web is appear music by year and load more than 5 results!!!every time load just 5 results and for albums and for songs!!!it would be great if you make it load more,less 50-100!!!Merry Christmas dude!!!

**Comment by zachomedia** [comment]
@kenpokas Yes, I will see if that limit can be increased. I assume you mean on the search pages? It may take me a few days to have some time to work on this due to the holidays.

**Comment by kenpokas** [comment]
woowwwwwwwww.....thanksssssssss so muchhhhhhhhhhhhhh dude!!!yes,I mean on the search of pages!!!my friend I hope you make it as soon as possible!!!pleaseeeeeeeeee I begging you!Zachomedia and if you can please and its not difficult for you make it appear also genre and date for every song!!!God bless you and your family also!!!happy new year my friend!!! (please tell me how you increase load unlimited resuts???)the other pages limit 200 songs only....congratulationsssssssssssssssssssss
